gef-ui/server/controllers/automatic-cover/index.js

```diff
@@ -39,10 +41,26 @@ const validateAutomaticCover = async (req, res, next) => {
     return receivedFields.some((field) => field === 'false');
   }
 
-  const { body, params } = req;
+  const { body, params, query } = req;
```

Comment:
Don't see query used anywhere?

Reply:
@davehlewis   It's used on line #46

Reply:
My bad - sorry not sure how I missed that!